Cargo.toml

```diff
@@ -44,7 +44,7 @@ thiserror = "1.0.30"
 htmlescape = "0.3.1"
 fail = { version = "0.5.0", optional = true }
 murmurhash32 = "0.3.0"
-time = { version = "0.3.10", features = ["serde-well-known"] }
+time = { version = "0.3.10", features = ["serde-well-known", "macros"] }
```

Comment:
why do do we need macros?

Reply:
I'm using the time! macro in the tests

Reply:
No test only dependencies as regular dependencies please

Reply:
Fair, moved the feature to dev-dependencies